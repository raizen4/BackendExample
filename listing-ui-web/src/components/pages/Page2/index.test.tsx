import React from 'react';
import { render } from '@testing-library/react';
import { BrowserRouter } from 'react-router-dom';
import Page from '../Page2';

describe('Home', () => {
  it('should render', () => {
    const { container } = render(
      <BrowserRouter>
        <Page />
      </BrowserRouter>
    );
    expect(container).toMatchSnapshot();
  });
});
