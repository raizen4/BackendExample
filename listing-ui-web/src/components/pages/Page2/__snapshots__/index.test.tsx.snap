// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home should render 1`] = `
<div>
  <div
    class="MuiContainer-root MuiContainer-maxWidthSm"
    style="opacity: 1; webkit-transition: opacity 225ms cubic-bezier(0.4, 0, 0.2, 1) 0ms; transition: opacity 225ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;"
  >
    <div
      class="makeStyles-wrapper-1"
    >
      <div
        class="MuiGrid-root MuiGrid-container MuiGrid-direction-xs-column MuiGrid-align-items-xs-center MuiGrid-justify-xs-center"
      >
        <h1
          class="MuiTypography-root MuiTypography-h1 MuiTypography-alignCenter"
        >
          Page 2
        </h1>
        <a
          href="/"
          style="text-decoration: none;"
        >
          <button
            class="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-containedSizeLarge MuiButton-sizeLarge"
            tabindex="0"
            type="button"
          >
            <span
              class="MuiButton-label"
            >
              GO to page 1
            </span>
            <span
              class="MuiTouchRipple-root"
            />
          </button>
        </a>
      </div>
    </div>
  </div>
</div>
`;
